doc/cli.markdown

```diff
@@ -1179,8 +1179,8 @@ This command can be used to start a shell on a local or remote device.
 If a service name is not provided, a shell will be opened on the host OS.
 
 If an application name is provided, all online devices in the application
-will be presented, and the chosen device will then have a shell opened on
-in it's service container or host OS.
+will be presented, and the chosen device will then have a shell opened
+in its service container or host OS.
```

Comment:
* `cli.markdown` is automatically generated. See https://github.com/balena-io/balena-cli/blob/master/CONTRIBUTING.md#editing-documentation-files-changelog-readme-website
  * The changes should be made to: https://github.com/balena-io/balena-cli/blob/v11.30.13/lib/actions/ssh.ts#L185-L187, followed by `npm run build` or `npm test`. Then amend/squash the commit and force push.

* I suggest a rewording beyond the grammar fix:
```
If an application name is provided, an interactive menu will be presented
for the selection of an online device. A shell will then be opened for the
host OS or service container of the chosen device.
```
(Mainly I disliked _"the chosen device will then have a shell opened".)_